src/validations/constraints/unit-tests/party-has-name-FAIL.yaml

```
@@ -0,0 +1,7 @@
+test-case:
+  name: Negative Test for party-has-name
+  description: This test case validates the behavior of constraint party-has-name
+  content: ../content/ssp-party-has-name-INVALID.xml
+  expectations:
+    - constraint-id: party-name
```

In case you go the route of changing the constraint id
```suggestion
    - constraint-id: party-has-name
```